corehq/apps/users/validation.py

```diff
@@ -11,14 +11,16 @@ def validate_mobile_username(username, domain, is_unique=True):
     Raises a ValidationError if any issue with the complete username is encountered
     :param username: str, expects complete username ('username@example.commcarehq.org')
     :param domain: str, required
+    :param is_unique: should be True when a new user is being created and false when a user is being updated.
```

Comment:
Thanks for adding, but going to request a small change. For the most part, the method itself shouldn't care about the context it will be used in, and I think that extends to docs. 

I think stating what `is_unique` achieves is more helpful than saying when a user of the method should set that value to true or false. Think about it as helping the user understand _why_ they would want to set `is_unique` to True or False, rather than telling them _when_ to do so.

Reply:
+1